src/sentry/models/integrations/sentry_app.py

```diff
@@ -191,11 +191,11 @@ def save(self, *args, **kwargs):
         self.date_updated = timezone.now()
         return super().save(*args, **kwargs)
 
-    def is_installed_on(self, organization):
+    def is_installed_on(self, *, organization_id: int):
```

Comment:
This is an example of one of many utility methods that would benefit from a `OrganizationWithId` type that represented, basically just the id of a thing that we want to query by.  It would allow reasonable compatibility with existing code and usage whilst also supporting the dataclasses.

Fortunately in this case, I was able to validate that there are only 3 callsites to this method that changing was no risk, so I did.